### DISCUSSION TITLE
Why are there so many extensions running in Activity Monitor?

### DISCUSSION THREAD

**Discussion by ghost** [aonez/Keka#1051] [Q&A] [answered]
![Ss 2022-02-23 at 19 39 32](https://user-images.githubusercontent.com/16494547/155285732-d75d6f08-c6bc-4db9-a2b7-d0598ba244b9.png)

Hi,

See screenshot. Why are there so many extensions running, not just one?

Thanks.

**Comment by gingerbeardman** [comment, marked as answer]
This is the macOS Finder way of managing things. It starts multiple for its own reasons, outside of the control of the app to whom the extensions belong. 

See #734 and please feel free to file a bug report with Apple.

**Reply by ghost** [reply]
Done. Thanks. :)

**Reply by gingerbeardman** [reply]
@aonez could you please remove the personal details on this page?

**Reply by aonez** [reply]
Done, thanks @gingerbeardman!

**Reply by ghost** [reply]
Sorry, gents. Had replied from email the first time.

**Reply by ghost** [reply]
![Ss 2022-02-24 at 06 15 17](https://user-images.githubusercontent.com/16494547/155396556-0d6ca2e5-80ac-424b-b755-7a73b30de24b.png)
FYI, Someone at www.macosicons.com made a very cool icon for Keka. If you like it, feel free to hit the designer up for it. I think it's awesome. :)

**Reply by aonez** [reply]
@swooned already saw that [a while ago](https://twitter.com/kekaosx/status/1481943223929081859?s=20&t=H4Ods6Hv5RgxwPgyoGszKg) but did not know who made it. On that website you point I don't see any way to contact the designer... Obviously I won't change the current icon from David, but I like this version as an alternate icon in the iOS version.

**Reply by ghost** [reply]
Yes. Not sure how to contact home either. If you want, you can email Elias who runs the site [an absolute blessing] to send you their details. I changed the main icon and the finder extension one, too, so they are consistent.

Thanks for making a clean compressor, unlike many others that are just so messy.

**Reply by ghost** [reply]
What does Keka mean?
